.env

```diff
@@ -1,4 +1,4 @@
 # Enviromental variables for docker-compose
 # Fill here your IP - do not use localhost or 127.0.0.1 - IoT Devices need real address in your local network
-#EXTERNAL_IP=10.100.0.150
-EXTERNAL_IP=10.100.0.150
+#EXTERNAL_IP= 192.168.1.34
+EXTERNAL_IP=192.168.1.33
```

Comment:
We should found the way, how to configure the demo without need to specify external IP.